Bottom: b19e7e7fde47ef8751a16a634f82bf401de0d341
Top:    324b9d62d185a0e316d91a6318645d292247140f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-30 16:08:43 +0100

Refresh of turn-off-autorouter-debug-draw

---

diff --git a/src/autoroute.c b/src/autoroute.c
index 416bc55..c99b58d 100644
--- a/src/autoroute.c
+++ b/src/autoroute.c
@@ -92,14 +92,16 @@ RCSID ("$Id$");
 #define ROUTE_VERBOSE
 */
 
+/*
 #define ROUTE_DEBUG
-#define DEBUG_SHOW_ROUTE_BOXES
+//#define DEBUG_SHOW_ROUTE_BOXES
 #define DEBUG_SHOW_EXPANSION_BOXES
-#define DEBUG_SHOW_EDGES
-#define DEBUG_SHOW_VIA_BOXES
+//#define DEBUG_SHOW_EDGES
+//#define DEBUG_SHOW_VIA_BOXES
 #define DEBUG_SHOW_TARGETS
 #define DEBUG_SHOW_SOURCES
-#define DEBUG_SHOW_ZIGZAG
+//#define DEBUG_SHOW_ZIGZAG
+*/
 
 static direction_t
 directionIncrement(direction_t dir)
@@ -4186,7 +4188,7 @@ RouteOne (routedata_t * rd, routebox_t * from, routebox_t * to, int max_edges)
       /* we should never add edges on inactive layer groups to the heap. */
       assert (is_layer_group_active[e->rb->group]);
 #if defined(ROUTE_DEBUG) && defined(DEBUG_SHOW_EXPANSION_BOXES)
-      showedge (e);
+      //showedge (e);
 #endif
       if (e->rb->flags.is_thermal)
 	{
